repository_service_tuf/helpers/cli.py

```diff
@@ -0,0 +1,138 @@
+# SPDX-License-Identifier: MIT
+
+import hashlib
+import os
+from dataclasses import asdict, dataclass
+from enum import Enum
+from typing import Any, Dict, List, Optional
+
+import click
+
+
+class PayloadTargetsHashes(str, Enum):
+    """The supported hashes of `TargetsInfo`"""
+
+    # TODO: Update if needed after https://github.com/repository-service-tuf/repository-service-tuf-api/issues/379  # noqa: E501
+
+    blake2b_256 = "blake2b-256"
```

Comment:
Now we support any hashes, but is good we choose one for rstuf cli.
In the future we can give options to user, but let's start simple.